server/server.js

```diff
@@ -1,56 +1,88 @@
 var express = require('express');
 var app = express();
 const db = require('./mysql/data.js');
+var bodyParser = require('body-parser');
 
-app.use(function(req, res, next) {
+/**
+ * body-parser 是必须要有的！ 不然request body是空的！
+ * Body-parser is the Node. js body parsing middleware.
+ * It is responsible for parsing the incoming request 
+ * bodies in a middleware before you handle it.
+ */
+
+app.use(function (req, res, next) {
     //add a header message to solve cors security policy access deny problem
     res.header("Access-Control-Allow-Origin", "http://localhost:3000"); // update to match the domain you will make the request from
     res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
+    res.header("Access-Control-Allow-Methods", "POST, GET, DELETE, OPTIONS");
     next();
-  });
+});
+
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json());
 
-app.get('/', async (req, res, next) =>{
-    console.log(req.body);
-    db.query('select * from master where contained_by = 0', [],function(result,fields){
+
+//home page's data
+app.get('/', async (req, res, next) => {
```

Comment:
Please consider to merge this with `/container/:id` get request handler method and leave it returning nothing.